documentation/library_guide/api_for_sycl_kernels_main.rst

```diff
@@ -13,9 +13,8 @@ API for the SYCL* Kernels
 .. toctree::
    :maxdepth: 2
    :titlesonly:
-   :glob:
    :hidden:
 
-   api_for_sycl_kernels/random
    api_for_sycl_kernels/tested_standard_cpp_api
-   api_for_sycl_kernels/utility_function_object_classes
\ No newline at end of file
+   api_for_sycl_kernels/random
+   api_for_sycl_kernels/utility_function_object_classes
```

Comment:
The topics are reordered to match the description at the page above, It makes more sense for the chain of "Next" links that are also generated from TOC.